code/basic-transactions/calc-tx-cost/calc-tx-est-fees-for-message.preview.en.ts

```diff
@@ -0,0 +1,33 @@
+const type = SYSTEM_INSTRUCTION_LAYOUTS.Transfer;
+const data = Buffer.alloc(type.layout.span);
+const layoutFields = Object.assign({instruction: type.index});
+type.layout.encode(layoutFields, data);
+
+const recentBlockhash = await connection.getLatestBlockhash();
+
+const messageParams = {
+  accountKeys: [
+      payer.publicKey.toString(),
+      payee.publicKey.toString(),
+      SystemProgram.programId.toString()
+  ],
+  header: {
+      numReadonlySignedAccounts: 0,
+      numReadonlyUnsignedAccounts: 1,
+      numRequiredSignatures: 1,
+  },
+  instructions: [
+      {
+      accounts: [0, 1],
+      data: bs58.encode(data),
+      programIdIndex: 2,
+      },
+  ],
+  recentBlockhash: recentBlockhash.blockhash,
+};
+
+const message = new Message(messageParams);
+
+const fees = await connection.getFeeForMessage(message);
```

Comment:
For this preview, it may be better to just have the

```
const message = new Message(messageParams);

const fees = await connection.getFeeForMessage(message);
```

Reply:
Okay, I'll simplify the preview.